modules/tinymce/src/plugins/autoresize/main/ts/api/Options.ts

```diff
@@ -26,24 +26,17 @@ const register = (editor: Editor): void => {
     processor: 'number',
     default: 50
   });
-
-  registerOption('autoresize_on_init', {
```

Comment:
Hmm, didn't know that we are going to remove this. There is no alternative for this at the moment. Other compeditors have a DIV mode that would act like this where the editor is the same size as the content. But I guess if it has been deprecated for a while then no one is using the editor like that?

Reply:
Not sure what you mean sorry Spocke, as this currently doesn't do that... due to the event handling we have the editor always resizes on init atm so this option in effect does nothing and that's why it's being removed.

Reply:
Hmm, it seems to still resize to the content based on the events. However not sure if this will work as it did before since the wait thing was to wait for style sheets to load and at the time you couldn't add a load event to the stylesheets on webkit and get it to fire. But that has been resolved since.

Reply:
Yes, I see now that other changes have been made to fire events when the content css files are loaded. So never mind. I tested it with a slow loading css file and seems to work fine.